osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs

```diff
@@ -55,6 +57,7 @@ public void ApplyToPlayer(Player player)
 
         public void Update(Playfield playfield)
         {
+            playfield.DisplayJudgements.Value = true;
```

Comment:
This is the only part which could be argued something we want, but many have stated they want the opposite of this, which is why it's implemented as it is right now.